easybuild/easyconfigs/j/JUBE/JUBE-2.0.3.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "VersionIndependentPythonPackage"
+
+name = "JUBE"
+version = "2.0.3"
+
+homepage = "http://www.fz-juelich.de/jsc/jube"
+description = """The JUBE benchmarking environment provides a script based framework to easily create benchmark sets,
+ run those sets on different computer systems and evaluate the results."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ["http://apps.fz-juelich.de/jsc/jube/jube2/download.php?version=%(version)s"]
```

Comment:
The problem is the automatic addition of  the tar.gz name at the end of the source url. Is there a way inside an easyconfig to skip this addition? https://apps.fz-juelich.de/jsc/jube/jube2/download.php?version=2.0.3 should work. Otherwise I will check if I can tweak the server side. At the moment it will read 2.0.3/JUBE-2.0.3.tar.gz as a version number, which doesn't exist and this will automatically fall back to the latest version.